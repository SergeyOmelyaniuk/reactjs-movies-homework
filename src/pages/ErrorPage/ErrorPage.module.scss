@import '../../scss/variables.scss';

.errorPage {
	padding: 100px 0px 120px 0px;
}

.wrapImage {
	max-width: 150px;
	width: 100%;
	height: 150px;
	margin-bottom: 30px;
}

.image {
	height: 100%;
	max-width: 100%;
	display: block;
	border-radius: 5px;
}

.title {
	font-size: 40px;
	color: $colorError;
	margin-bottom: 20px;
}

.text {
	font-size: 20px;
}
