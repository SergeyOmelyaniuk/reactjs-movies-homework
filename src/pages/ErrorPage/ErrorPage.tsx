import React from 'react';
import styles from './ErrorPage.module.scss';

function ErrorPage() {
	return (
		<div className={styles.errorPage}>
			<div className={styles.wrapImage}>
				<img
					className={styles.image}
					src={require(`../../assets/images/error.png`).default}
					alt=''
				/>
			</div>
			<div className={styles.title}>Something went wrong</div>
			<div className={styles.text}>Please, reload the page or try later</div>
		</div>
	);
}

export default ErrorPage;
