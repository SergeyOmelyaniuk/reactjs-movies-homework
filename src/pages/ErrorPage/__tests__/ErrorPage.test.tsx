import React from 'react';
import { render } from '@testing-library/react';
import ErrorPage from '..';

describe('ErrorPage', () => {
	test('should render', () => {
		const { container } = render(<ErrorPage />);

		expect(container).toMatchSnapshot();
	});
});
