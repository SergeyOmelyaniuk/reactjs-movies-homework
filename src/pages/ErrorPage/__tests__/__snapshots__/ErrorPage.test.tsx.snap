// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage should render 1`] = `
<div>
  <div
    class="errorPage"
  >
    <div
      class="wrapImage"
    >
      <img
        alt="error"
        class="image"
      />
    </div>
    <div
      class="title"
    >
      Something went wrong
    </div>
    <div
      class="text"
    >
      Please, reload the page or try later
    </div>
  </div>
</div>
`;
